tests/transports/test_linkietransport.py

```diff
@@ -0,0 +1,82 @@
+import base64
+
+import pytest
+
+from kasa.credentials import DEFAULT_CREDENTIALS, Credentials, get_default_credentials
+from kasa.deviceconfig import DeviceConfig
+from kasa.exceptions import KasaException
+from kasa.transports.linkietransport import LinkieTransportV2
+
+KASACAM_REQUEST_PLAINTEXT = '{"smartlife.cam.ipcamera.dateTime":{"get_status":{}}}'
+KASACAM_RESPONSE_ENCRYPTED = "0PKG74LnnfKc+dvhw5bCgaycqZOjk7Gdv96syaiKsJLTvtupwKPC7aPGse632KrB48/tiPiX9JzDsNW2lK6fqZCgmKuZoZGh3A=="
+KASACAM_RESPONSE_ERROR = '{"smartlife.cam.ipcamera.cloud": {"get_inf": {"err_code": -10008, "err_msg": "Unsupported API call."}}}'
+
+
+async def test_working(mocker):
+    transport_no_creds = LinkieTransportV2(config=DeviceConfig("127.0.0.1"))
+    mocker.patch.object(
+        transport_no_creds._http_client,
+        "post",
+        side_effect=post_custom_return(200, KASACAM_RESPONSE_ENCRYPTED),
+    )
+    # Expected response
+    # '{"timezone": "UTC-05:00", "area": "America/New_York", "epoch_sec": 1690832800}'
+    response = await transport_no_creds.send(KASACAM_REQUEST_PLAINTEXT)
+    assert response["epoch_sec"] == 1690832800
```

Comment:
```suggestion
    response = await transport_no_creds.send(KASACAM_REQUEST_PLAINTEXT)
    assert response == {"timezone": "UTC-05:00", "area": "America/New_York", "epoch_sec": 1690832800}
```
You can test against the complete dictionary and drop the comment :-)

Reply:
Why didn't that occur to me :upside_down_face: